packages/arb-ts/scripts/deployStandard.ts

```diff
@@ -0,0 +1,131 @@
+import { instantiateBridge } from './instantiate_bridge'
+import { ERC20__factory } from '../src/lib/abi/factories/ERC20__factory'
+import dotenv from 'dotenv'
+import args from './getCLargs'
+import { constants, BigNumber, utils } from 'ethers'
+
+dotenv.config()
+
+const privKey = process.env.PRIVKEY as string
+if (!privKey) {
+  throw new Error('Set PRIVKEY env variable')
+}
+
+if (!args.l1_token_address) {
+  throw new Error(
+    'Include l1_token_address arg (--l1_token_address 0xmytokenaddress)'
+  )
+}
+if (!args.amount) {
+  throw new Error('Include amount arg (--amount 123)')
+}
+
+const { l1_token_address: l1TokenAddress, amount } = args
+const amountBigNum = BigNumber.from(amount)
+
+const main = async () => {
+  const { bridge } = await instantiateBridge(privKey, privKey)
+  const walletAddress = await bridge.l1Bridge.getWalletAddress()
+
+  const l1Token = ERC20__factory.connect(l1TokenAddress, bridge.l1Provider)
+  // TODO: check blacklist
```

Comment:
We could add a disclaimer about what isn't bridgeable and a link to the docs and force the user to say "yes" they have read / are aware / know what they are doing etc